torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -108,17 +108,14 @@ def resize_image_tensor(
     extra_dims = image.shape[:-3]
 
     if image.numel() > 0:
-        resized_image = _FT.resize(
+        image = _FT.resize(
             image.view(-1, num_channels, old_height, old_width),
             size=[new_height, new_width],
             interpolation=interpolation.value,
             antialias=antialias,
         )
-    else:
-        # TODO: the cloning is probably unnecessary. Review this together with the other perf candidates
-        resized_image = image.clone()
 
-    return resized_image.view(extra_dims + (num_channels, new_height, new_width))
+    return image.view(extra_dims + (num_channels, new_height, new_width))
```

Comment:
We do not care if it is an inplace op for empty tensor, right ?

Reply:
Depending on your definition of "inplace" I don't think this is an inplace op. For me "inplace" means, the returned tensor has the same `.data_ptr()` but it's values have changed. Since a degenerate input has no data to begin with, the data cannot be changed in any way. Thus, I think there are no inplace operation for degenerate tensors in general. What did you have in mind?

Reply:
Oh, sorry, I was thinking view is an inplace op, my thoughts was that `x.view(n, c, new_h, new_w)` will also update x.